### DISCUSSION TITLE
OpenCore Legacy Patcher issue with Mac pro 2013

### DISCUSSION THREAD

**Discussion by Joel-Hache** [community/community#143761] [Programming Help]
### Select Topic Area

Question

### Body

Hi,
I'm new to this site, so maybe i didn't post the question to the right place. I urgently need help with the OCLP.
I'm using a Mac Pro 2013 (Trashcan), with the latest MacOS, i've used the latest OCLP version to install (2.1.0).
My main use is Adobe apps, Indesign and Photoshop. When i try to upgrade to the latest version of those apps, i get the error that this update requires AVX2, which apparently i don't have.
Is there any way for configuring OCLP for adding this support? Or any other solution (except for buying a new mac).
Thanks a lot.
Joel.

**Comment by CleverLemming1337** [comment]
Is your Mac early or late 2013?

**Comment by Joel-Hache** [comment]
Mac Pro, late 2013.

**Reply by CleverLemming1337** [reply]
OK, then you have a Haswell CPU which should generally support AVX2.

**Comment by Joel-Hache** [comment]
I don't know, anyway when trying to upgrade my Adobe products, i get the message that it requires support for avx2.
So, what can i do about it?

I've checked the System info, it says: 2.7 GHz 12-Core Intel Xeon E5.

**Reply by CleverLemming1337** [reply]
Intel Xeon is Sandy Bridge, which should support AVX2. If it says it requires support, you cannot install it, right?

**Comment by Joel-Hache** [comment]
Right, so if it does support avx2, is there anything i can do to make this installation work? This is my main app i'm using all the time.
Thanks.

**Reply by CleverLemming1337** [reply]
I'm sorry, but I have no other ideas. I'm no real expert at this.

**Comment by Joel-Hache** [comment]
Thanks.
